input/templates/my-account-change-password.json

```diff
@@ -2,8 +2,9 @@
 	"header" : { "$ref" : "common/header.json" },
 	"footer" : { "$ref" : "common/footer.json" },
 	"content" : {
-		"changePassword": {
-			"feedbackMessage": "Password successfully changed."
+		"changePasswordForm": {
```

Comment:
I have renamed `changePassword` to `changePasswordForm`. It's a good practice to put always the `Form` suffix in the data that contains form data, so that we distinguish between the info coming from our platform and a form data. Here is not that clear, but if you think about addresses, `address` would be the address coming from our platform, and `addressForm` would be the form to, for example, edit an address (in this case is even better to call it `editAddressForm` to distinguish between edit and create actions).